### ISSUE TITLE
Tile frequency used in loss function

### ISSUE THREAD

**Issue by schrum2** [schrum2/MarioDiffusion#231]
To account for tile frequency imbalance, use inverse tile frequency as a component of the cross-entropy loss function
